scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -811,7 +812,7 @@ task IsUsingCompressedReferences {
       SELECT
         column_name
       FROM
-        `~{dataset_name}.INFORMATION_SCHEMA.COLUMNS`
+        `~{dest_project_id}.~{dataset_name}.INFORMATION_SCHEMA.COLUMNS`
```

Comment:
lovely

Reply:
actually I don't know wdl. Is this syntax correct?

Reply:
Why is `query_project_id` being introduced here if it's never used? Possibly related, line 811 is still referencing `project_id` which no longer exists.

`womtool validate` will check the syntactical correctness of WDL for you, and is among the actions that execute automatically against `ah_var_store` PRs in this repo. I'm not sure why it didn't flag this error. 🤔

Reply:
Ah, good finding! I don't have full context about this, this is just a sample PR to explain my finding. 
In our scenario, user has bqJobUser role on `query_project_id` - Terra workspace project. User does not have permission on dest_project.